src/wayland-eglsurface.c

```diff
@@ -1599,7 +1587,6 @@ resize_callback(struct wl_egl_window *window, void *data)
                                    surface,
                                    surface,
                                    pData->egl.getCurrentContext());
-
```

Comment:
Line can be added back to shrink the diff

Reply:
Done, thanks.